cli/config/configuration-fc.toml.in

```diff
@@ -78,6 +79,9 @@ default_memory = @DEFMEMSZ@
 # Default 0
 #memory_offset = 0
 
+# Specify whether cpu/memory used by guest is counted in container resource quota.
+#overhead_in_quota = false
```

Comment:
Resource quota is a kubernetes admission controller -- is that what you're referring to in this name?

Reply:
No, not really, naming is the hardest part, do you have some suggestion?